dcnm/resource_dcnm_policy.go

```diff
@@ -401,3 +395,21 @@ func deploySwitchFabric(dcnmClient *client.Client, serialNumber string) error {
 
 	return nil
 }
+
+func recurSwitchDeployment(dcnmClient *client.Client, serialNumber string) {
+	err := deploySwitchFabric(dcnmClient, serialNumber)
```

Comment:
We should add a debug log that we are deploying a specific switch. We can add the log here or in the deploySwitchFabric function.

Reply:
Is this fixed in the parent function?